Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/LiveSpyNavPage.xaml.cs

```diff
@@ -102,13 +102,13 @@ private void SetFrameEnableDisable()
         /// This method is used to get the new WindowExplorerPage based on Context and Agent
         /// </summary>
         /// <returns></returns>
-        private void LoadWindowExplorerPage(Context context)
+        private void LoadWindowExplorerPage()
         {
```

Comment:
why we have 'LoadWindowExplorerPage()' method in LiveSpy page?

Reply:
I've just updated the method signature removing context parameter but haven't visited the code. I'll go through this once and revert.